components/rating/index.css

```diff
@@ -96,13 +94,11 @@ governing permissions and limitations under the License.
 
   /* When the entire component is hovered, show all solid icons */
   &:hover {
-    .spectrum-Rating-icon {
-      .spectrum-Rating-starActive {
-        display: block;
-      }
-      .spectrum-Rating-starInactive {
-        display: none;
-      }
+    .spectrum-Rating-starActive {
```

Comment:
Yeah just to avoid the max depth error I removed it since there wasn't any styling specific to the `spectrum-Rating-icon` class and there isn't any instance of these being outside of it